kits/democracy/test/democracy.js

```diff
@@ -20,7 +20,13 @@ const getContract = name => artifacts.require(name)
 
 const pct16 = x => new web3.BigNumber(x).times(new web3.BigNumber(10).toPower(16))
 const getEventResult = (receipt, event, param) => receipt.logs.filter(l => l.event == event)[0].args[param]
-const createdVoteId = receipt => getEventResult(receipt, 'StartVote', 'voteId')
+const getVoteId = (receipt) => {
```

Comment:
Does `getEventResult()` not work for getting this? I was under the impressions truffle would parse the logs for us.

Reply:
No, it was not working, because it's not the "first" call (now we are forwarding trough `TokenManager`), so logs don't show up in `receipt.logs.filter` and we have to look for them in `receipt.receipt.logs` instead.